### PR TITLE
DOCU-3314 Post-GA release of API Product Builder spec

### PR DESCRIPTION
**Pull request by acgoldsmith**
### Description

Split from https://github.com/Kong/docs.konghq.com/pull/5760

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Looks like this is causing the broken link checker to fail, but I'm guessing that it's because the spec isn't published yet, not because the URL is wrong.

**Comment by acgoldsmith** [comment]
@sichvoge , this PR is ready to merge for tomorrow's release. Please ensure the spec is live at https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0 ahead of merging this PR. Thanks!

File: app/konnect/updates.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
-
 ## July 2023
 
+**API Products API released**
+: The API products API has been released to support the objective of making API consumption straightforward for developers. Explore the API spec on our [Dev Portal](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0)
```
Comment (sichvoge) [inline app/konnect/updates.md:15]:
```suggestion
: A new Konnect API for managing API products and versions is now available for external consumption. This API allows you to create and manage API products and versions, upload documentation and spec, and link a version to an existing Gateway service to enable application registration. As a result, you can easily integrate this API into your automated pipeline to streamline publishing documentation for your products to your third party developers. Explore the API spec on our [Developer Portal](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0)
```

**Review by sichvoge — CHANGES_REQUESTED** [review]
Adjust release notes to match mine.

File: app/konnect/updates.md [hunk 2]
```diff
@@ -9,9 +9,11 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
-
 ## July 2023
 
+**API Products API released**
+: A new Konnect API for managing API products and versions is now available for external consumption. This API allows you to create and manage API products and versions, upload documentation and spec, and link a version to an existing Gateway service to enable application registration. As a result, you can easily integrate this API into your automated pipeline to streamline publishing documentation for your products to your third party developers. Explore the API spec on our [Developer Portal](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0)
```
Comment (cloudjumpercat) [inline app/konnect/updates.md:15]:
```suggestion
: A new {{site.konnect_short_name}} API for managing API products and versions is now available for external consumption. This API allows you to create and manage API products and versions, upload documentation and specs, and link a version to an existing Gateway service to enable application registration. As a result, you can integrate this API into your automated pipeline to streamline publishing documentation for your products to your third-party developers. Explore the API spec on our [Developer Portal](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0)
```

**Pull request merged**